docs/source/operators/config-security.md

```diff
@@ -152,54 +152,58 @@ The list of [additional supported environment variables](config-add-env.md#addit
 Enterprise Gateway supports Secure Sockets Layer (SSL) communication with its clients. With SSL enabled, all the
 communication between the server and client are encrypted and highly secure.
 
-1.  You can start Enterprise Gateway to communicate via a secure protocol mode by setting the `certfile` and `keyfile`
-    options with the command:
+1. You can start Enterprise Gateway to communicate via a secure protocol mode by setting the `certfile` and `keyfile`
+   options with the command:
 
-        ```
-        jupyter enterprisegateway --ip=0.0.0.0 --port_retries=0 --certfile=mycert.pem --keyfile=mykey.key
-        ```
+   ````
```

Comment:
I'm guessing these 4 back tics are needed to prevent hard-coded section numbers, but it doesn't render the embedded CLI examples (and their surrounding text) correctly.  Actually, when looking at the current docs, I see the same kind of output - so this change is not at fault.  Do you know how to go about making this rendor as expected so that the triple-back tics are applied?

![image](https://user-images.githubusercontent.com/22599560/196242017-3e697ba4-b0fc-46f2-bb83-97716519d2dc.png)

hmm.  If I remove the quad-back tics, leaving the numbering as is, I see the content of bullet 1 rendered correctly:
![image](https://user-images.githubusercontent.com/22599560/196244265-387c2baa-86d6-4c02-872c-8575b69b1d5c.png)

As a result, I think we should remove the quad-tics.